src/jukebox/jukebox/multitimer.py

```diff
@@ -249,6 +249,7 @@ def start(self, iterations=None, wait_seconds_per_iteration=None):
         self._function = self._callee(*self.class_args, iterations=self._iterations, **self.class_kwargs)
         super().start(wait_seconds_per_iteration)
 
+    @plugin.tag
```

Comment:
Why?

Reply:
https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2112 
Hasn't to do anything with venv, but was a noop fix.